### DISCUSSION TITLE
Visualization of grain orientations

### DISCUSSION THREAD

**Discussion by tianjuxue** [neperfepx/neper#383] [Q&A] [answered]
Hello,

How do you produce the plot **in the middle** (the one showing cubes with different orientations)? 

![Screen Shot 2022-02-07 at 1 41 56 PM](https://user-images.githubusercontent.com/45647025/152860287-ed80acd9-647f-4e3b-8e03-8c0b4c093af8.png)
 
Thank you,
Tianju

**Comment by rquey** [comment, marked as answer]
You can use `-showcrystal`, `-datacrystal`, etc.

Example:
```
neper -T -n 200 -oricrysym cubic
neper -V n100-id1.tess -print cells -showcell 0 -showcrystal 1 -showedge "domtype==1" -print crystals
```
![image](https://user-images.githubusercontent.com/2659376/156525331-f27de1bb-ee29-4bb0-9162-040514ddd0ae.png)
![image](https://user-images.githubusercontent.com/2659376/156525336-6fd29ee8-424a-4632-a66a-a023ceab4e4e.png)

See also https://neper.info/tutorials/visualize_tessellation.html.
